modules/platforms/dotnet/Apache.Ignite.Core/Client/Services/IServicesClient.cs

```diff
@@ -38,6 +40,19 @@ public interface IServicesClient
         /// <returns>Proxy object that forwards all member calls to a remote Ignite service.</returns>
         T GetServiceProxy<T>(string serviceName) where T : class;
 
+        /// <summary>
+        /// Gets metadata about all deployed services in the grid.
+        /// </summary>
+        /// <returns>Metadata about all deployed services in the grid.</returns>
+        ICollection<IClientServiceDescriptor> ServiceDescriptors();
```

Comment:
```suggestion
        ICollection<IClientServiceDescriptor> GetServiceDescriptors();
```

(to be consistent with `IServices` and .NET naming guidelines)

Reply:
Fixed